ldi-core/sparql-construct/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/SparqlConstructTransformer.java

```diff
@@ -70,4 +73,11 @@ private List<Model> splitDataset(Dataset dataset) {
 		return result;
 	}
 
+	private void initGeoFunctions() {
+		FunctionRegistry.get().put(FirstCoordinate.name, FirstCoordinate.class);
```

Comment:
I don't know how much code gets triggered with every "FunctionRegistry.get()" but it could be an idea to call it once at the top of the method and then just do "functionRegistry.put(...)" every time.